tests/integration/test_application.py

```diff
@@ -103,7 +103,7 @@ def app(create_app):
         (["-h"], "", BASIC_USAGE, 0),
         (["--version"], VERSION_INFO, "", 0),
         (["-V"], VERSION_INFO, "", 0),
-        (["-q", "--version"], VERSION_INFO, "", 0),
+        (["-q", "--version"], "", "", 0),
```

Comment:
This actually showed an inconsistency: `rockcraft --quiet --version` outputs the version but `rockcraft --quiet version` does not